<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="3">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>638</width>
         <height>588</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QFormLayout" name="objectsLayout"/>
        </item>
        <item row="1" column="0">
         <layout class="QFormLayout" name="attributesLayout"/>
        </item>
        <item row="2" column="0">
         <layout class="QFormLayout" name="pluginsLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QFormLayout" name="annotationsLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1014</width>
         <height>588</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <layout class="QGridLayout" name="PlayerLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1680</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentProjects">
     <property name="title">
      <string>Recent Projects</string>
     </property>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="menuRecentProjects"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionClasses"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithm">
    <property name="title">
     <string>Algorithm</string>
    </property>
    <addaction name="actionAuto_Annotate"/>
    <addaction name="actionCompress_Session"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAlgorithm"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="../../resource/icons.qrc">
     <normaloff>:/Icons/open-or.png</normaloff>:/Icons/open-or.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset resource="../../resource/icons.qrc">
     <normaloff>:/Icons/save-or.png</normaloff>:/Icons/save-or.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="icon">
    <iconset resource="../../resource/icons.qrc">
     <normaloff>:/Icons/save-or.png</normaloff>:/Icons/save-or.png</iconset>
   </property>
   <property name="text">
    <string>Save Project As ...</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="icon">
    <iconset resource="../../resource/icons.qrc">
     <normaloff>:/Icons/delet-or.png</normaloff>:/Icons/delet-or.png</iconset>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="../../resource/icons.qrc">
     <normaloff>:/Icons/edit-or.png</normaloff>:/Icons/edit-or.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resource/icons.qrc">
     <normaloff>:/Icons/quit-or.png</normaloff>:/Icons/quit-or.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAuto_Annotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Annotate</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About &amp;Annotator</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>.</string>
   </property>
  </action>
  <action name="actionDs">
   <property name="text">
    <string>ds</string>
   </property>
  </action>
  <action name="actionClasses">
   <property name="text">
    <string>Classes</string>
   </property>
  </action>
  <action name="actionCompress_Session">
   <property name="text">
    <string>Compress Session</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resource/icons.qrc"/>
 </resources>
 <connections/>
</ui>
